functionnal_tests/cypress/integration/activity_feed/activity_spec.js

```diff
@@ -47,6 +47,7 @@ describe('At the space recent activities page', () => {
       cy.get('[data-cy=commentArea__comment__send]').click()
     })
     it('should display the file preview', () => {
+      cy.wait(2000)
```

Comment:
ow
it hurts : (
we'll have to at it

Reply:
Yes, I'm currently working on it! (and noticed I shouldn't do this anymore ^^')

Reply:
ok ^^
I can help you if you need, I'll be way more available starting from today